.actor/input_schema.json

```diff
@@ -0,0 +1,27 @@
+{
+	"title": "Omio scraper",
+	"type": "object",
+	"schemaVersion": 1,
+	"properties": {
+		"from": {
+			"title": "From",
+			"description": "From location.",
+			"editor": "textfield",
+			"type": "string"
+		},
+		"to": {
+			"title": "To",
+			"description": "To location.",
+			"editor": "textfield",
+			"type": "string"
+		},
+		"date": {
+			"title": "Date",
+			"type": "string",
+			"description": "Select an absolute travel date in format YYYY-MM-DD. The date must be either today or in the future (past dates are not supported).",
+			"editor": "datepicker",
+			"pattern": "^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[12]\\d|3[01])$"
```

Comment:
You can also allow the relative datepicker, see https://docs.apify.com/platform/actors/development/actor-definition/input-schema/specification/v1#string